scripts/live-connector-e2e/run-windows.ps1

```
@@ -591,6 +591,30 @@ function Test-BlockVerdict([string]$Path, [int]$Since) {
+function Wait-GatewayEvidenceAfter(
+    [string]$Path,
+    [string]$Name,
+    [int]$Since,
+    [bool]$RequireBlock,
+    [int]$TimeoutMilliseconds = 5000
+) {
+    $deadline = [DateTime]::UtcNow.AddMilliseconds($TimeoutMilliseconds)
+    $connectorEvent = $false
+    $blockVerdict = $false
+    do {
+        $connectorEvent = Test-ConnectorEvent $Path $Name $Since
```

PSScriptAnalyzer (1.25.0)

[info] Cmdlet 'Test-ConnectorEvent' has positional parameter. Please use named parameters instead of positional parameters when calling a command.

(PSAvoidUsingPositionalParameters)